pkgs/build-support/dotnet/make-nuget-deps/default.nix

```diff
@@ -4,7 +4,7 @@ linkFarmFromDrvs "${name}-nuget-deps" (nugetDeps {
   fetchNuGet = { pname, version, sha256
     , url ? "https://www.nuget.org/api/v2/package/${pname}/${version}" }:
     fetchurl {
-      name = "${pname}-${version}.nupkg";
+      name = "${pname}.${version}.nupkg";
```

Comment:
Why was this done?

Reply:
This makes the structure match e.g. `Private.SourceBuilt.Artifacts.*.tar.gz`, which is used directly as a source when building the sdk.

There's also a `fetchNuGet` in `github-runner` and `fetchnuget` with both use name.version.

Reply:
If it matches upstream dotnet more closely that's fine by me :+1: